### DISCUSSION TITLE
Mobile/Touch Support?

### DISCUSSION THREAD

**Discussion by kenaliento** [atlassian/pragmatic-drag-and-drop#93] [Q&A]
Is this library well suited for mobile/touch devices? 

After trying it out in Svelte, I realized I needed to test it on touch. However, no luck. It wasn't usable. Same with the examples in the documentation. It would start "dragging" the element but only 10% it would allow me to drop it somewhere. Also the press + hold time frame is too long. It would be awesome if there is a way to change that on touch/mobile.

**Comment by tehilamehudar** [comment]
I’m experiencing the same issue. On touch devices the press-and-hold duration feels noticeably long (longer than what I’m used to with react-beautiful-dnd). Is there currently any way to adjust or shorten that time frame?
